src/Microsoft.Identity.Client/ApiConfig/Parameters/AcquireTokenCommonParameters.cs

```diff
@@ -14,11 +14,11 @@ internal class AcquireTokenCommonParameters
 
         public ApiEvent.ApiIds ApiId { get; set; } = ApiEvent.ApiIds.None;
         public Guid TelemetryCorrelationId { get; set; }
+        public Guid UserProvidedTelemetryCorrelationId { get; set; }
```

Comment:
>Telemetry [](start = 32, length = 9)

This is not necessarily a telemetry id... It's a CorrelationId which has been user provided. Pls. remove the word Telemetry